scripts/render.py

```diff
@@ -5,7 +5,9 @@
 import argparse
 import numpy as np
 
-sys.path.append("/home/pranav/dev/celldiv/scripts")
+#Default: violet stiff (healthy), orange soft (ill)
+
+sys.path.append("/Users/Torsa_Legend/Desktop/ROBE TESI/Progr/scripts")
```

Comment:
Ah, we have to change this line. It shouldn't be required anymore since the compilation should add it to `PYTHONPATH`